pom.xml

```diff
@@ -243,6 +243,32 @@
 				</repository>
 			</repositories>
 		</profile>
+		<profile>
+			<id>licence-check</id>
+			<pluginRepositories>
+				<pluginRepository>
+					<id>dash-licenses-snapshots</id>
+					<url>https://repo.eclipse.org/content/repositories/dash-licenses-snapshots/</url>
+					<snapshots>
+						<enabled>true</enabled>
+					</snapshots>
+				</pluginRepository>
+			</pluginRepositories>
+			<build>
+				<plugins>
+					<plugin>
+						<groupId>org.eclipse.dash</groupId>
+						<artifactId>license-tool-plugin</artifactId>
+						<version>0.0.1-SNAPSHOT</version>
+						<configuration>
+							<failWhenReviewNeeded>true</failWhenReviewNeeded>
+							<!-- TODO: maven/mavencentral/org.openjdk.jmh/jmh-generator-annprocess/1.35, GPL-2.0-only WITH Classpath-exception-2.0, restricted, #3045 -->
+							<excludeGroupIds>org.openjdk.jmh</excludeGroupIds>
```

Comment:
This is fine to put in as an exception: we have previously gotten this dependency reviewed by the IP team. Since we only include it for use in our own testing, and don't distribute it, it doesn't require further review.